lib/assets/stylesheets/chef/components/top-bar.scss

```diff
@@ -82,145 +35,109 @@ Add the `skinny` modifier to get a thinner bar more appropriate for apps or smal
+  The Top Bar component also requires a JavaScript module in order for the
+  mobile menu dropdown to work properly. Be sure to include
+  chef.js in your page, along with whatever additional scripts you might need. See the
   <a href="/getting-started">Getting Started</a> section for more information.
 </div>
 
-## More Information
-  * [Foundation Top Bar documentation](http://foundation.zurb.com/docs/components/topbar.html)
 */
 
-@import '../settings';
-@import '../base/typography';
-@import '../themes/default';
-@import '../layouts/default';
-@import 'button';
-@import 'logo';
-
-$topbar-bg-color:                   $chef-white !default;
-$topbar-bg:                         $topbar-bg-color !default;
-$topbar-dropdown-bg:                $topbar-bg !default;
-$topbar-logo-margin:                10px !default;
-$topbar-height:                     rem-calc($logo-height + ($topbar-logo-margin * 2)) !default;
-$topbar-link-color:                 scale-color($chef-dark-blue, $lightness: 10%) !default;
-$topbar-link-color-active:          $topbar-link-color !default;
-$topbar-link-color-active-hover:    $topbar-link-color !default;
-$topbar-link-color-hover:           scale-color($topbar-link-color, $lightness: 20%) !default;
-$topbar-link-bg-hover:              $topbar-bg !default;
-$topbar-link-bg-color-hover:        $topbar-bg !default;
-$topbar-link-font-family:           $chef-type-label !default;
-$topbar-link-font-size:             16px !default;
-$topbar-link-padding:               $topbar-height / 8 !default;
-$topbar-link-text-transform:        uppercase !default;
-$topbar-link-weight:                bold !default;
-$topbar-button-height:              rem-calc($topbar-link-font-size) + ($topbar-link-padding * 2);
-$topbar-menu-link-color:            $topbar-link-color !default;
-$topbar-menu-icon-color:            $topbar-link-color !default;
-$topbar-menu-link-font-size:        rem-calc(13) !default;
-$topbar-menu-link-transform:        uppercase !default;
-$topbar-menu-link-weight:           normal !default;
-$topbar-margin-bottom:              $paragraph-margin-bottom !default;
-
-@import 'foundation/components/top-bar';
-
-@mixin top-bar($scale: 1, $height: $topbar-height, $link-font-size: $topbar-link-font-size) {
-  $scaled-height: $height * $scale;
-  
-  max-height: $scaled-height;
-  transition: max-height .4s ease-in;
-
-  .title-area {
-    height: $scaled-height;
-
-    .name {
-      h1 {
-        line-height: $scaled-height;
-
-        a.logo {
-          width: $logo-width-vertical * $scale;
-          height: $logo-height * $scale;
-          margin-top: $topbar-logo-margin * $scale;
-          margin-left: ($column-gutter / 2) * $scale;
-          padding: 0;
-
-          &.horizontal {
-            width: $logo-width-horizontal * $scale;
-          }
-        }
-      }
-    }
+@import "chef/settings";
+
+.top-bar {
+  line-height: 1;
+  margin-bottom: 20px;
+  min-height: 50px;
+  padding: 11px 0.9375rem;
```

Comment:
To make it match what I was seeing on downloads:

![screenshot](https://dl.dropboxusercontent.com/s/7wg828y3z84tg3k/2015-11-09%20at%204.42%20PM.png)

The markup is different, but I wanted it to be as close as possible.